evaluator/logic.py

```diff
@@ -530,15 +531,27 @@ async def _evaluate_advisor_res(
         for cve, hit_details in rule_results["rule_hits"].items():
             cve_id = await self._get_or_upsert_cve(cve)
 
+            # TODO: remove this once CVE_2017_5753_4_cpu_kernel and CVE_2017_5715_cpu_virt are merged
             rule = hit_details["rule_id"]
+            if rule in self.skipped_rules:
```

Comment:
Also, this is the remaining part, from the previous `advisor_listener` because I thought it got fixed and merged, but it did not :)

Reply:
Maybe we could ask the rule team to permanentl delete these rules

Reply:
I agree